src/libraries/System.Private.CoreLib/src/System/Numerics/Vector4.Extensions.cs

```diff
@@ -3,27 +3,43 @@
 
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
 
 namespace System.Numerics
 {
     public static unsafe partial class Vector
     {
+        /// <summary>Reinterprets a <see cref="Vector4" /> as a new <see cref="Plane" />.</summary>
+        /// <param name="value">The vector to reinterpret.</param>
+        /// <returns><paramref name="value" /> reinterpreted as a new <see cref="Plane" />.</returns>
+        internal static Plane AsPlane(this Vector4 value)
+        {
+#if MONO
```

Comment:
Is this perf problem or functionality problem?

Either way, it is likely going to be fixed as side-effect of https://github.com/dotnet/runtime/issues/102988 .

Reply:
Possibly a functionality problem that I've not dug into yet. The previous tested commit had seen some failures related to Quaternion/Vector4/Plane and their equality tests, but only on Mono.

My current guess is that there's something subtly incorrect in the Mono handling that breaks for SIMD here and it will need a fix before BitCast can be used, but I'd like to try and ensure CoreCLR is clean without regressions before I do any more in depth Mono changes.